scripts/sql/231_devtron_CD_trigger_v1.0.0.up.sql

```diff
@@ -0,0 +1,247 @@
+INSERT INTO plugin_metadata (id,name,description,type,icon,deleted,created_on,created_by,updated_on,updated_by)
+VALUES (nextval('id_seq_plugin_metadata'),'The plugin enables users to trigger pre/post/deployment of application. It helps users deploy the application that contains dependencies for their current application.','PRESET','https://raw.githubusercontent.com/devtron-labs/devtron/main/assets/devtron-logo.png',false,'now()',1,'now()',1);
+
+
+INSERT INTO plugin_stage_mapping (id,plugin_id,stage_type,created_on,created_by,updated_on,updated_by)VALUES (nextval('id_seq_plugin_stage_mapping'),
+(SELECT id from plugin_metadata where name='Devtron CD Trigger v1.0.0'), 0,'now()',1,'now()',1);
+
+
+INSERT INTO "plugin_pipeline_script" ("id", "script","type","deleted","created_on", "created_by", "updated_on", "updated_by")
+VALUES ( nextval('id_seq_plugin_pipeline_script'),
+E'#!/bin/bash
+
+# Convert CD Workflow Type to uppercase to make it case-insensitive
+TargetTriggerStage=$(echo "$TargetTriggerStage" | tr \'[:lower:]\' \'[:upper:]\')
+
+# Set default value for TargetTriggerStage to DEPLOY if not provided or invalid
+case $TargetTriggerStage in
+"PRE")
+    ;;
+"POST")
+    ;;
+"DEPLOY")
+    ;;
+"")
+    TargetTriggerStage="DEPLOY" # Set to DEPLOY if no input provided
+    ;;
+*)
+    echo "Error: Invalid CD Workflow Type. Please enter PRE, DEPLOY, or POST. Exiting..."
```

Comment:
change it to - Error: Invalid CD Workflow Type. Please enter PRE/POST/DEPLOY. Exiting...